setup.cfg

```diff
@@ -23,9 +23,14 @@ previous = true
 
 [nosetests]
 rednose=1
+hide-skips=1
+immediate=1
```

Comment:
Yes, as you can see in the latest build no more skipped test reported as error. In case of error, `immediate` prevent us to wait for all tests to finish